rules/network/command_and_control_cobalt_strike_default_teamserver_cert.toml

```
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2020/10/05"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/05"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the use of the default Cobalt Strike Team Server TLS certificate. Cobalt Strike is software for
+Adversary Simulations and Red Team Operations which are security assessments that replicate the tactics and techniques
+of an advanced adversary in a network. If using Filebeat, this rule requires the Suricata or Zeek modules. Modifications
+to the Packetbeat configuration can be made to include MD5 and SHA256 hashing algorithms (the default is SHA1) - see the
+Reference section for additional information on module configuration.
+"""
+index = ["filebeat-*", "packetbeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Default Cobalt Strike Team Server Certificate"
+note = "While Cobalt Strike is intended to be used for penetration tests and IR training, it is frequently used by actual threat actors (TA) such as APT19, APT29, APT32, APT41, FIN6, DarkHydrus, CopyKittens, Cobalt Group, Leviathan, and many other unnamed criminal TAs. This rule uses high-confidence atomic indicators, alerts should be investigated rapidly."
+references = [
+    "https://attack.mitre.org/software/S0154/",
+    "https://www.cobaltstrike.com/help-setup-collaboration",
+    "https://www.elastic.co/guide/en/beats/packetbeat/current/configuration-tls.html",
+    "https://www.elastic.co/guide/en/beats/filebeat/7.9/filebeat-module-suricata.html",
+    "https://www.elastic.co/guide/en/beats/filebeat/7.9/filebeat-module-zeek.html",
+]
+risk_score = 100
+rule_id = "e7075e8d-a966-458e-a183-85cd331af255"
+severity = "critical"
+tags = [
+    "Command and Control",
+    "Post-execution",
```

```suggestion
    "Post-Execution",
```

Minor tweak required for test to pass.